Example/Example/AppDelegate.swift

```diff
@@ -123,14 +124,25 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
             CLLocation(latitude: 40.64189, longitude: -73.78779),
             CLLocation(latitude: 35.99801, longitude: -78.94294)
         ]
-        
+
         Radar.getMatrix(origins: origins, destinations: destinations, mode: .car, units: .imperial) { (status, matrix) in
             print("Matrix: status = \(Radar.stringForStatus(status)); matrix[0][0].duration.text = \(String(describing: matrix?.routeBetween(originIndex: 0, destinationIndex: 0)?.duration.text)); matrix[0][1].duration.text = \(String(describing: matrix?.routeBetween(originIndex: 0, destinationIndex: 1)?.duration.text)); matrix[1][0].duration.text = \(String(describing: matrix?.routeBetween(originIndex: 1, destinationIndex: 0)?.duration.text)); matrix[1][1].duration.text = \(String(describing: matrix?.routeBetween(originIndex: 1, destinationIndex: 1)?.duration.text))")
         }
 
+
+        Radar.sendEvent(name: "app_launched", metadata: nil) { (status, location, events, user) in
+            print("Send event: status = \(Radar.stringForStatus(status)); location = \(String(describing: location)); events = \(String(describing: events)); user = \(String(describing: user))")
+        }
+
+        Radar.getLocation { (status, location, stopped) in
```

Comment:
maybe add a comment here that this is testing `sendEvent` with a manual location?